src/tests/unittests/algorithms/test_functions.py

```diff
@@ -62,6 +61,11 @@ def test_logistic():
         y = logistic(0.0, 1.0, alpha=1.0)
         np.testing.assert_almost_equal(y, 0.11920292202211755, decimal=6)
 
+    @staticmethod
+    def test_gaussian():
+        y = gaussian(np.array([0.0, 1., 2.]), 1.0, .2)
+        np.testing.assert_almost_equal(y, np.array([3.72665317e-06, 1.00000000e+00, 3.72665317e-06]), decimal=6)
```

Comment:
This test doesn't use the unit test framework to test and return a result. This causes exception when running this test individually.

There are more of these tests in this file.

Reply:
Please have a look at the tests in sub directory integration\algorithms. Maybe test_fitting and test_functions here need some import changes also.

Reply:
> This test doesn't use the unit test framework to test and return a result. This causes exception when running this test individually.
> 
> There are more of these tests in this file.

Can you explain this? The tests run for me, and I have no option to run tests individually.

Reply:
@eendebakpt I use pycharm where each unit test method or test class can be run separately from the IDE. This is essential for coding and testing because I don't want to run all the test all the time.

Reply:
Fixed this by making the method not-static. Since then codacy is complaining about `this method could be static`  I added a test which uses the object (although it does not test much)